app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -20,6 +20,9 @@
         v-on-clickaway="hideEmojiPicker"
         :on-click="emojiOnClick"
       />
+      <reply-email-head
+        @set-emails="setCcEmails"
```

Comment:
@fayazara / @nithindavid After this change I am able to get the value of the set CC and BCC emails in this component from child component but now the validations aren't working.

Reply:
@tejaswinichile Please try this.

Replace the v-model values with `v-model.trim="$v. ccEmailsVal.$model"`. This will proxy the v-models through the vuelidate library and also, in the `onBlur()` method. You need to add a check to proceed only if its valid.

Use this.$v.invalid 
```
if (this.$v.$invalid) {
       // Handle Invalid data
} else {
      // Proceed
}
``` 

More on vuelidate lib [here](https://vuelidate.js.org/#sub-basic-form)

Reply:
@fayazara Thanks, That worked 👍🏼